ddp-workspace/projects/ddp-dsm-ui/src/app/services/dsm.service.ts

```diff
@@ -231,6 +231,48 @@ export class DSMService {
     );
   }
 
+  public downloadParticipantData(realm: string, parent: string, columns: {}, json: ViewFilter, filterQuery: string, sortBy?: Sort): Observable<any> {
```

Comment:
the whole method body looks like copy/paste of `applyFilter`, is it not possible to avoid this duplication by extracting method or something like that?